DuckDuckGo/AutofillLoginSettingsListViewController.swift

```diff
@@ -40,8 +40,9 @@ final class AutofillLoginSettingsListViewController: UIViewController {
     weak var delegate: AutofillLoginSettingsListViewControllerDelegate?
     weak var detailsViewController: AutofillLoginDetailsViewController?
     private let viewModel: AutofillLoginListViewModel
-    private let emptyView = AutofillItemsEmptyView()
+    private lazy var emptyView = AutofillItemsEmptyView()
```

Comment:
Is there a particular reason to change this one to lazy and not any of the others?

Reply:
The others are all immediately added as subviews as part of `viewDidLoad` -> `installSubviews` so didn't see any real benefit to making them lazy, while this view is now only added as a subview in one specific view state (`empty`)